reporting/impl/src/main/resources/reports/templates/include/navheader.ftl

```diff
@@ -0,0 +1,9 @@
+            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-responsive-collapse">
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            <span class="wu-navbar-header">
+              <strong class="wu-navbar-header">Red Hat Application Migration Toolkit</strong>
+               - Web Console 
```

Comment:
Yep, you're right, the report comes from CLI even if it's opened from Web UI.
So, the official name to put in the bar is "CLI" instead of "Web Console", right?